avm/res/api-management/service/README.md

```diff
@@ -1532,39 +1551,6 @@ A list of availability zones denoting where the resource needs to come from.
 
 _None_
 
-## Notes
```

Comment:
Just doublecheckinf if this was intended. You're more than welcome to remove these notes that originally came from CARML. Just want to make sure you did know about them 😉